x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -853,3 +853,171 @@ A conscious effort has been made to mask this complexity with sane defaults and
 documentation above but in case you encounter issues while configuring a SAML realm, you can
 look through our {stack-ov}/trb-security-saml.html[SAML troubleshooting documentation] that has
 suggestions and resolutions for common issues.
+
+=== SAML without {kib}
```

Comment:
If we don't specify an anchor here, our build makes one up, which is not reliable. I suggest:
```suggestion
[[saml-no-kibana]]
=== SAML without {kib}
```